spec/minidoc/timestamps_spec.rb

```diff
@@ -20,5 +22,27 @@ class TimestampsUser < Minidoc
       user.save
       expect(user.created_at).to_not eq user.updated_at
     end
+
+    it "updates updated_at when doc is changed with #set" do
+      user = TimestampsUser.create
+      sleep 0.001
+      user.set(name: "Abby Normal")
+      expect(user.created_at).to_not eq user.updated_at
+    end
+
+    it "updates updated_at when doc is changed with .atomic_set" do
```

Comment:
Ah, sure.

I think the discrepencay is fine -- I mentioned it just in case you hadn't considered it, but clearly you have -- but we should probably mention that either in the README or CHANGELOG. If we really intend those class methods only to be helpers, I'm +1 on making them private in 2.0.0